icu4c/source/common/normalizer2impl.cpp

```diff
@@ -2088,6 +2088,10 @@ uint8_t Normalizer2Impl::getPreviousTrailCC(const uint8_t *start, const uint8_t
 // minDecompNoCP etc. and smallFCD[] are intended to help with any loss of performance,
 // at least for ASCII & CJK.
 
+// Ticket 20907 - Disable optimization on Windows when building for ARM64 with Visual Studio versions below 16.4.
```

Comment:
In this comment, could you add an approximate timeframe during which this workaround is required, such that if someone comes around and refactors/rewrites this function, they know when they can delete it?

Reply:
Thanks! That's a good idea. I'll expand the comment and force push.